compiler/src/iree/compiler/Codegen/Common/MaterializeEncodingPass.cpp

```diff
@@ -256,16 +257,36 @@ chooseDynamicEncodingInfoVMVXMicrokernels(RankedTensorType tensorType,
                                           OpBuilder &builder, Location loc) {
   Optional<TensorEncoding> encoding = getEncoding(tensorType);
   if (!encoding) return failure();
+  auto matmulType = getMatmulType(*encoding);
```

Comment:
FYI @hanhanW seems like we will need to have a CPUMaterializeEncoding pass and a VMVXMaterializeEncoding pass.

Reply:
I'm moving the pass to LLVMCPU/ and can send out a draft later today. We can split it into CPUMaterializeEncoding pass and VMVXMaterializeEncoding pass after that.